website-generator/resources/src/table_search.js

```diff
@@ -4,18 +4,28 @@
 
 // Table search
 (function () {
-  function onSearch (inputId, tableId) {
+  function onSearch(inputId, tableId, tableType) {
     const input = document.getElementById(inputId);
     const filter = input.value.toLowerCase();
     const table = document.getElementById(tableId);
     const testNames = table.getElementsByClassName('testName');
     const allBackendsResults = table.getElementsByClassName('backendResults');
+    const showPassed = document.getElementById('showPassed' + tableType).checked;
+    const showFailed = document.getElementById('showFailed' + tableType).checked;
+    let testStatus = '';
 
+    if (showPassed && !showFailed) {
+      testStatus = 'passed';
+    }
+    else if (showFailed && !showPassed) {
+      testStatus = "failed";
```

Comment:
Let's stick to one convention here.
```suggestion
      testStatus = 'failed';
```